### PR TITLE
chore: Storybook implementation

### PR DESCRIPTION
**Pull request by castastrophe**
## Description

- Add storybook with button & helptext example
- Add global css to storybook

Commands added:

`yarn start`: Builds components & spins up storybook; opens localhost in browser
`SKIP_BUILD=true yarn start`: Sneaky way to suppress the pre-build components step.

https://pr-1539--spectrum-css.netlify.app/preview/?path=/story/button--default

## How and where has this been tested?
 - **How this was tested:** 
 - **Browser(s) and OS(s) this was tested with:** 

## To-do list

- [x] If my change impacts other components, I have tested to make sure they don't break.
- [ ] If my change impacts documentation, I have updated the documentation accordingly.
- [x] I have read the [CONTRIBUTING document](/.github/CONTRIBUTING.md).
- [x] I have tested these changes in Windows High Contrast mode.

- [ ] This pull request is ready to merge.

### CODE REVIEW

File: tools/preview/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# @spectrum-css/preview
```
Comment (castastrophe) [inline tools/preview/README.md:1]: Todo: need to fill this out

**Review by pfulton — CHANGES_REQUESTED** [review]
Wow, this is awesome @castastrophe. 🎉 

My one bit of feedback at this time is that we should get rid of the option for `lightest`, as we've deprecated it.

File: tools/preview/decorators/contextsWrapper.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
+import { useEffect } from '@storybook/client-api';
+
+import '@spectrum-css/vars/dist/spectrum-global.css';
+import '@spectrum-css/vars/dist/components/index.css';
+
+import '@spectrum-css/vars/dist/spectrum-lightest.css';
```
Comment (castastrophe) [inline tools/preview/decorators/contextsWrapper.js:6]:
```suggestion
```

File: tools/preview/decorators/contextsWrapper.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        document.body.classList.add('spectrum--express');
+      } else {
+        stylesheets.forEach(style => style.unuse());
+        document.body.classList.remove('spectrum--express');
+      }
+    }
+    
+    if (window.__spectrum_context__.color !== color) {
+      window.__spectrum_context__.color = color;
+      ['lightest', 'light', 'dark', 'darkest'].forEach(c => document.body.classList.remove(`spectrum--${c}`));
```
Comment (castastrophe) [inline tools/preview/decorators/contextsWrapper.js:58]:
```suggestion
      ['light', 'dark', 'darkest'].forEach(c => document.body.classList.remove(`spectrum--${c}`));
```

File: tools/preview/preview.js [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+  },
+};
+
+// Global properties added to each component;
+//      determines what stylesheets are loaded
+export const argTypes = {
+    color: {
+      name: 'Color',
+      description: 'Controls the color context of the component.',
+      options: ['lightest', 'light', 'dark', 'darkest'],
```
Comment (castastrophe) [inline tools/preview/preview.js:58]:
```suggestion
      options: ['light', 'dark', 'darkest'],
```

File: tools/preview/preview.js [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+      defaultValue: 'light',
+      type: { required: true },
+      table: { category: "Global", defaultValue: { summary: 'light' } },
+      control: {
+        type: 'select',
+        labels: {
+          lightest: 'Lightest',
```
Comment (castastrophe) [inline tools/preview/preview.js:65]:
```suggestion
```

**Review by castastrophe — COMMENTED** [review]
Removing lightest via GitHub UI

**Pull request merged**